### DISCUSSION TITLE
Allure report opens "This site can’t be reached" page

### DISCUSSION THREAD

**Discussion by NataliiaStetsyk** [allure-framework/allure2#1936] [Bug Reports] [answered]
Hi! 
I try to generate allure report, but it opens an error page like this: 
"This site can’t be reached The web page at http://192.168.88.7:50979/ might be temporarily down or it may have moved permanently to a new web address.
ERR_SOCKET_NOT_CONNECTED" 
![image](https://user-images.githubusercontent.com/34649004/222891218-345dff84-4fb0-4d55-9266-0dc966c1d4ca.png)
![image](https://user-images.githubusercontent.com/34649004/222891246-12fab3ae-d552-4887-8f90-6fd6487599e6.png)

Could you please help how to fix this? 
Thanks!

**Comment by cheshi-mantu** [comment, marked as answer]
Hi @NataliiaStetsyk 
try to use the allure command something like this
```shell
allure serve path/to/allure-results --host localhost --port 9999
```
or if you've already generated the report, then

```shell
allure open path/to/allure-report-files --host localhost --port 9999
```

**Reply by NataliiaStetsyk** [reply]
Thanks, it works!
